Library/Homebrew/cmd/info.rb

```diff
@@ -179,6 +179,20 @@ def info_formula(f)
 
     caveats = Caveats.new(f)
     ohai "Caveats", caveats.to_s unless caveats.empty?
+
+    return if ENV["HOMEBREW_NO_ANALYTICS"]
+    return if ENV["HOMEBREW_NO_GITHUB_API"]
+    output, = curl_output("https://formulae.brew.sh/api/formula/#{f}.json")
```

Comment:
Should this be `output, _` so it looks less like a typo and more like an intentionally ignored return component?